nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/settings/ui/flow-analysis-rules/flow-analysis-rule-table/flow-analysis-rule-table.component.html

```diff
@@ -0,0 +1,145 @@
+                    <ng-container *ngIf="canRead(item)">
+                        {{ formatType(item) }}
+                    </ng-container>
+                </td>
+            </ng-container>
+
+            <!-- Bundle Column -->
+            <ng-container matColumnDef="bundle">
+                <th mat-header-cell *matHeaderCellDef mat-sort-header>Bundle</th>
+                <td mat-cell *matCellDef="let item">
+                    <ng-container *ngIf="canRead(item)">
+                        {{ formatBundle(item) }}
+                    </ng-container>
+                </td>
+            </ng-container>
+
+            <!-- State Column -->
+            <ng-container matColumnDef="state">
+                <th mat-header-cell *matHeaderCellDef mat-sort-header>State</th>
+                <td mat-cell *matCellDef="let item">
+                    <div class="flex items-center gap-x-2">
+                        <div [ngClass]="getStateIcon(item)"></div>
+                        <div>{{ formatState(item) }}</div>
+                        <div *ngIf="hasActiveThreads(item)">({{ item.status.activeThreadCount }})</div>
+                    </div>
+                </td>
+            </ng-container>
+
+            <!-- Actions Column -->
+            <ng-container matColumnDef="actions">
+                <th mat-header-cell *matHeaderCellDef></th>
+                <td mat-cell *matCellDef="let item">
+                    <div class="flex items-center gap-x-3">
+                        <div
+                            class="pointer fa fa-gear"
+                            *ngIf="canConfigure(item)"
+                            (click)="configureClicked(item, $event)"
+                            title="Edit"></div>
+                        <!-- TODO - handle read only in configure component? -->
+                        <div
+                            class="pointer fa icon icon-enable-false"
+                            *ngIf="canDisable(item)"
+                            (click)="disableClicked(item, $event)"
+                            title="Disable"></div>
+                        <div
+                            class="pointer fa fa-flash"
+                            *ngIf="canEnable(item)"
+                            (click)="enabledClicked(item, $event)"
+                            title="Enable"></div>
+                        <div class="pointer fa fa-exchange" *ngIf="canChangeVersion(item)" title="Change Version"></div>
+                        <div
+                            class="pointer fa fa-trash"
+                            *ngIf="canDelete(item)"
+                            (click)="deleteClicked(item)"
+                            title="Delete"></div>
+                        <div class="pointer fa fa-tasks" *ngIf="canViewState(item)" title="View State"></div>
+                    </div>
+                    <div class="pointer fa fa-key" *ngIf="canManageAccessPolicies()" title="Access Policies"></div>
```

Comment:
I don't believe that FlowAnalysisRules supports granular access policies.

Reply:
We do however need to handle State and Change Version but we can handle that when we introduce those capabilities. We have line items for them in NIFI-12400.

Reply:
ok I think I mixed up the Enforcement policy with access policy. I can remove this.